src/app/pages/user/dialogs/add/add-user.dialog.html

```diff
@@ -59,9 +59,9 @@ <h3 mat-dialog-title>הוספת משתמש חדש</h3>
               <mat-error *ngIf="EmailFormControl.hasError('email')">אימייל לא חוקי</mat-error>
                 </mat-form-field>
               </div>
-              <div class="msw-form-row">
+              <div clSass="msw-form-row">
                   <mat-form-field color="primary">
-                    <mat-select placeholder="כיתה" formControlName="class" [(ngModel)]="data.Class" [(value)]="data.Class" name="class" [disabled]="currentRole !== userRoleEnum.TEACHER">
+                    <mat-select placeholder="כיתה" formControlName="class" [(ngModel)]="data.Class" [(value)]="data.Class" name="class" [disabled]="isManager()">
```

Comment:
Maybe instead it's better to create service that had generic function `isInRole(role: Role Type)`?

Reply:
not sure how this is DRY. I removed the method and just added comparison in HTML